platforms/android/tangram/src/main/java/com/mapzen/tangram/MapController.java

```diff
@@ -772,330 +772,139 @@ public void setRenderMode(@IntRange(from=0,to=1) final int renderMode) {
     }
 
     /**
-     * Set a responder for tap gestures
-     * @param responder TapResponder to call
-     */
-    public void setTapResponder(@Nullable final TouchInput.TapResponder responder) {
-        touchInput.setTapResponder(new TouchInput.TapResponder() {
-            @Override
-            public boolean onSingleTapUp(final float x, final float y) {
-                return responder != null && responder.onSingleTapUp(x, y);
-            }
-
-            @Override
-            public boolean onSingleTapConfirmed(final float x, final float y) {
-                return responder != null && responder.onSingleTapConfirmed(x, y);
-            }
-        });
-    }
-
-    /**
-     * Set a responder for double-tap gestures
-     * @param responder DoubleTapResponder to call
-     */
-    public void setDoubleTapResponder(@Nullable final TouchInput.DoubleTapResponder responder) {
-        touchInput.setDoubleTapResponder(new TouchInput.DoubleTapResponder() {
-            @Override
-            public boolean onDoubleTap(final float x, final float y) {
-                return responder != null && responder.onDoubleTap(x, y);
-            }
-        });
-    }
-
-    /**
-     * Set a responder for long press gestures
-     * @param responder LongPressResponder to call
+     * Get the {@link TouchInput} for this map.
+     *
+     * {@code TouchInput} allows you to configure how gestures can move the map. You can set custom
+     * responders for any gesture type in {@code TouchInput} to override or extend the default
+     * behavior.
+     *
+     * Note that {@code MapController} assigns the default gesture responders for the pan, rotate,
+     * scale, and shove gestures. If you set custom responders for these gestures, the default
+     * responders will be replaced and those gestures will no longer move the map. To customize
+     * these gesture responders and preserve the default map movement behavior: create a new gesture
+     * responder, get the responder for that gesture from the {@code MapController}, and then in the
+     * new responder call the corresponding methods on the {@code MapController} gesture responder.
+     * @return The {@code TouchInput}.
      */
-    public void setLongPressResponder(@Nullable final TouchInput.LongPressResponder responder) {
-        touchInput.setLongPressResponder(new TouchInput.LongPressResponder() {
-            @Override
-            public void onLongPress(final float x, final float y) {
-                if (responder != null) {
-                    responder.onLongPress(x, y);
-                }
-            }
-        });
+    public TouchInput getTouchInput() {
+        return touchInput;
     }
 
     /**
-     * Set a responder for pan gestures
-     * @param responder PanResponder to call; if onPan returns true, normal panning behavior will not occur
+     * Get a responder for pan gestures
```

Comment:
Even though the above documentation for `TouchInput` mention that these responders provide the default implementation. I would still prefer the documentation for these explicitly mention "default - map movement behavior".